pkgs/by-name/ll/llama-cpp/package.nix

```
@@ -146,7 +156,7 @@ effectiveStdenv.mkDerivation (finalAttrs: {
     license = licenses.mit;
     mainProgram = "llama-cpp-main";
```

Upstream picks `llama` as the main program.
```suggestion
    mainProgram = "llama";
```